opte/src/engine/mod.rs

```diff
@@ -123,7 +124,7 @@ cfg_if! {
 
             unsafe {
                 if opte_debug != 0 {
-                    let cstr = cstr_core::CString::new(msg).unwrap();
+                    let cstr = CString::new(msg).unwrap();
```

Comment:
Not really part of this PR, but we might want to truncate the message or something, rather than unwrapping.

Reply:
IIRC, the unwrap is only in case this is invalid UTF-8, and I see no way for that to ever be the case given that all these should be strings coming from opte code or from network implementations using opte code. I.e., I can't ever see this string coming from bytes passed down by the user. But in any event, truncation does make sense in terms of line length. We really never should be making long debug/failure messages. And so we could probably replace this with something like a static 1024 byte buffer like we've talked about for the SDT probes.